### DISCUSSION TITLE
Is there an issue search function?

### DISCUSSION THREAD

**Discussion by x1y9** [gogs/gogs#7271] [Q&A]
I've noticed a few PRs, but the version I installed myself and the official online demo don't have a search function for issues.  I think this should be a basic function. If I have dozens or more issues, no search is basically unusable.

**Comment by paulej** [comment]
I was looking for this today.  I was looking for something akin to what `git grep` would produce.  Anything more substantial could result in a pretty complex search index (both in size and management overhead).
